release-notes/release-notes54.md

```diff
@@ -30,6 +30,8 @@ The {{site.productname}} 5.4 release includes the following improvements for the
 - Adds: commands, APIs, and icons for; cut, copy, and paste columns.
 - Adds toolbar button icons for the cut, copy, and paste rows.
 - Adds a new `mceTableApplyCellStyle`command for applying selected styles to table cells.
+- Adds a new `table_header_type` option for setting the structure used for table header rows.
```

Comment:
"HTML structure" to be slightly more specific?

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1590/commits/966fedd38b68c8897d01d21eefeb077e14ea954c